resolwe/flow/models/entity.py

```diff
@@ -10,6 +9,12 @@
 from .base import BaseModel
 from .collection import BaseCollection
 
+if settings.USE_TZ:
```

Comment:
This pattern is used a lot. It would be nice to move it to `resolwe.utils` and import it from there. But it can be done in a separate pull request.

Reply:
Good point, but let's do that in in a separate effort. I'll write it down.